sdk/databox/Microsoft.Azure.Management.DataBox/AzSdk.RP.props

```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
   <PropertyGroup>
-    <AzureApiTag>DataBox_2018-01-01;</AzureApiTag>
+    <AzureApiTag>DataBox_2018-01-01;DataBox_2019-09-01;</AzureApiTag>
```

Comment:
Is there any reason you don't want `DataBox_2018-01-01;` to be removed?

Reply:
It can be removed. I'll fix it